base-template/doctype-example-guide.md

```diff
@@ -4,7 +4,10 @@ This guide explains how to build a good example document for a specific `doctype
 
 The guide is designed to be read by a template author, someone who intends to create or improve a set of templates and guides for a `doctype`.
 
+{
+Add structured metadata info to your doc template that the end user can use to describe the doc page's contents, to improve search results that feature the doc pages. If your docs are to be displayed in HTML, add a script tag to the head of the page with JSON-LD metadata, as in this example: 
```

Comment:
```suggestion
* Add [machine readable structured metadata](https://developers.google.com/search/docs/guides/sd-policies) to your doctype example.
* Structured metadata helps search engines index pages appropriately, and facilitates content reuse.
* Within this project we have adopted the [JSON-LD](http://json-ld.org/) format for structured metadata, [as recommended by Google](https://developers.google.com/search/docs/guides/intro-structured-data).
```
I've copied similar wording from the base-doctype-template, to retain consistency.